ui/scripts/configuration.js

```diff
@@ -2415,7 +2415,7 @@
                                             $supportedServices.css('display', 'inline-block');
                                         } else if ($guestTypeField.val() == 'L2') {
                                             $useVpc.hide();
-                                            $supportedServices.hide();
+                                            $supportedServices.css('display', 'inline-block');
```

Comment:
@rhtyd what if we display a checkbox with label 'User data (ConfigDrive)'? If it is checked, we can set serviceProviderList parameter to ConfigDrive and if not we'll pass it blank. This way we avoid user selecting unsupported services from the UI. What do you think?

Reply:
Good idea, perhaps you or Ernie can improve the UI changes.

Reply:
Sure, I can pick it up during the weekend

Reply:
Cool, Monday is fine too.